charts/drag-provider/package.json

```diff
@@ -23,9 +23,13 @@
   },
   "dependencies": {
     "@dnd-kit/core": "^6.3.1",
-    "@dnd-kit/sortable": "^10.0.0"
+    "@dnd-kit/sortable": "^10.0.0",
+    "@lg-charts/core": "workspace:^"
```

Comment:
we could get around this by moving the chart drag tests to charts-core. That may be more "technically correct" since the weirdness is in Charts